packages/node_modules/@node-red/nodes/core/function/10-function.html

```diff
@@ -70,12 +129,54 @@
                 }
             });
 
+            this.initEditor = RED.editor.createEditor({
+                id: 'node-input-init-editor',
+                mode: 'ace/mode/nrjavascript',
+                value: $("#node-input-initialize").val(),
+                globals: {
```

Comment:
Can we define the globals object once and reuse for each of the three editors? Would reduce the risk of them getting out of sync if we make an update to one but not the others.

Reply:
merged globals definitions.